src/monitor-configurations/monitor-configurations.service.ts

```diff
@@ -109,17 +113,17 @@ export class MonitorConfigurationsService {
     queryTime: string,
   ): Promise<LastUpdatedConfigDTO> {
     const dto = new LastUpdatedConfigDTO();
-
-    const clock: Date = (await this.entityManager.query('SELECT now();'))[0]
+    const slaveQueryRunner = this.entityManager.connection.createQueryRunner("slave");
+    const clock: Date = (await slaveQueryRunner.query('SELECT now();'))[0]
       .now;
     dto.mostRecentUpdate = clock;
```

Comment:
The query runner needs to be released after use:
```suggestion
    const slaveQueryRunner = this.entityManager.connection.createQueryRunner("slave");
    slaveQueryRunner.connect();
    try {
      const clock: Date = (await slaveQueryRunner.query('SELECT now();'))[0]
        .now;
      dto.mostRecentUpdate = clock;
    } finally {
      slaveQueryRunner.release();
    }
```

Reply:
updated this in latest commit